models/chunter/src/index.ts

```diff
@@ -177,6 +178,32 @@ export class TDirectMessageInput extends TClass implements DirectMessageInput {
   component!: AnyComponent
 }
 
+@Model(chunter.class.ChatMessage, activity.class.ActivityMessage, DOMAIN_CHUNTER)
+export class TChatMessage extends TActivityMessage implements ChatMessage {
+  @Prop(TypeMarkup(), chunter.string.Message)
+  @Index(IndexKind.FullText)
+    message!: string
+
+  @Prop(PropCollection(attachment.class.Attachment), attachment.string.Attachments, {
+    shortLabel: attachment.string.Files
+  })
+    attachments?: number
+
+  @Prop(TypeBoolean(), core.string.Boolean)
+  @Index(IndexKind.Indexed)
```

Comment:
nit: indexes on boolean attributes usually does not make sense because of low ccardinality

Reply:
Thanks, removed